fe/fe-core/src/main/java/org/apache/doris/catalog/Type.java

```diff
@@ -659,6 +659,36 @@ public Integer getColumnSize() {
         }
     }
 
+    /**
+     * For schema change, convert data type to string,
+     * get the size of string representation
+     */
+    public Integer getColumnStringRepSize() {
```

Comment:
I think the BE side also needs to have a self-defense mechanism to ensure that when the FE does not judge the length correctly, it will not cause the BE to crash?